src/test/java/com/cdancy/bitbucket/rest/BaseBitbucketApiLiveTest.java

```diff
@@ -67,12 +70,28 @@ protected Properties setupProperties() {
         return overrides;
     }
 
-    protected synchronized String getDefaultUser() {
-        if (defaultUser == null) {
+    protected synchronized String getDefaultUserAsString() {
+        if (defaultUserAsString == null) {
             if (this.credential.contains(":")) {
-                defaultUser = this.credential.split(":")[0];
+                defaultUserAsString = this.credential.split(":")[0];
             } else {
-                defaultUser = new String(base64().decode(this.credential)).split(":")[0];
+                defaultUserAsString = new String(base64().decode(this.credential)).split(":")[0];
+            }
+        }
+        return defaultUserAsString;
+    }
+
+    protected synchronized User getDefaultUser() {
```

Comment:
I like this better. Would you mind just removing the `defaultUserAsString` and we will standardize on just using the `User` object across the board. All classes can then just query the `User` object for its actual name.